static/css/index.css

```
@@ -0,0 +1,797 @@
+            font-weight: 700;
+        }
+
+        .logs {
+            background: rgba(8, 12, 16, 0.7);
+            border: 1px solid rgba(255, 255, 255, 0.08);
+            border-radius: 10px;
+            padding: 10px 12px;
+            height: calc(100vh - 250px);
+            min-height: 260px;
+            overflow-y: auto;
+            font-family: "Cascadia Mono", "Consolas", monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "Consolas" (font-family-name-quotes)

(font-family-name-quotes)